src/locale/mr.js

```diff
@@ -52,19 +89,19 @@ export default moment.defineLocale('mr', {
         sameElse : 'L'
     },
     relativeTime : {
-        future : '%s नंतर',
-        past : '%s पूर्वी',
-        s : 'सेकंद',
-        m: 'एक मिनिट',
-        mm: '%d मिनिटे',
-        h : 'एक तास',
-        hh : '%d तास',
-        d : 'एक दिवस',
-        dd : '%d दिवस',
-        M : 'एक महिना',
-        MM : '%d महिने',
-        y : 'एक वर्ष',
-        yy : '%d वर्षे'
+        future: '%sमध्ये',
```

Comment:
Do you think it would make more fluid if this was '%sत' ?
Examples would be -
काही सेकंदांत
५ दिवसांत
एका मिनिटात

I am fine with either, I just think 'मध्ये' sounds more like spoken Marathi rather than written Marathi. 

Your call.